Vision/Vision.bs

```diff
@@ -38,9 +38,9 @@ Markup Shorthands: markdown yes
 
 	* Help the world understand what W3C is, what it does, and why it matters
 	* Communicate shared values and principles of the W3C community
-	* Be opinionated enough to provide a framework for making decisions,
+	* Be opinionated enough to provide guidance and a framework for making decisions,
 		particularly on controversial issues
-	* Be timeless enough to guide W3C yet flexible enough to evolve when needed
+	* Be timeless enough to remain relevant without needing frequent revision
```

Comment:
no strong view on relevant vs dependable.

For the second half of the sentence, I wonder if the problem is comes from "timeless" being a quality of the content of document, while "flexible" being rather a property of the way we maintain it. In other words, if we did have a commitment to maintain the document and evolve it as needed, could we make the document itself inflexible, defeating that intent to maintain?
This makes me think that the original ("without needing…") may be closer to what we need.

Reply:
Fixing the STOD might be a better solution to this:
```diff
- It will continue to evolve, and the AB will issue updates as often as needed.
+ It will continue to evolve and the AB will issue updates as needed,
+ though such updates are not expected to be frequent.
```
(together with keeping the "Be timeless enough to remain relevant without needing frequent revision" phrasing in the document itself)

Reply:
I don't believe so. It was good enough for Ethical Web Principles:
>  It will continue to evolve and the TAG will issue updates as often as needed.

in https://w3ctag.github.io/ethical-web-principles/ to go to Statement so I'd rather keep our own wording the same without a very good reason to diverge.